resources/views/maps/markers/index.blade.php

```diff
@@ -0,0 +1,44 @@
+<?php /** @var \App\Models\Map $model */?>
+@extends('layouts.app', [
+    'title' => __('maps.maps.title', ['name' => $model->name]),
+    'description' => '',
+    'breadcrumbs' => false,
+    'mainTitle' => false,
+])
+
+@inject('campaignService', 'App\Services\CampaignService')
+
+@section('entity-header-actions')
+    @can('update', $model)
+        <div class="header-buttons">
+            <a href="https://docs.kanka.io/en/latest/entities/maps/layers.html" class="btn btn-default btn-sm" target="_blank">
+                <i class="fa-solid fa-question-circle" aria-hidden="true"></i> {{ __('crud.actions.help') }}
+            </a>
+            @if ($model->explorable())
+                <a href="{{ route('maps.explore', ['map' => $model]) }}" class="btn btn-primary btn-sm">
+                    <i class="fa-solid fa-map" aria-hidden="true"></i>
+                    {{ __('maps.actions.explore') }}
+                </a>
+            @endif
+        </div>
+    @endcan
+@endsection
+
+@section('content')
+    <div class="entity-grid">
+        @include('entities.components.header', [
+            'model' => $model,
+            'breadcrumb' => [
+                ['url' => Breadcrumb::index('maps'), 'label' => __('maps.index.title')],
+                null
```

Comment:
Missing the "Markers" breadcrumb. Same for Layers and Groups. See the Abilities or INventory subpage for what I mean
![image](https://user-images.githubusercontent.com/3263440/197286536-819b1c27-7757-4c46-b1a7-0fe4d57df9fe.png)